product_management/README.md

```diff
@@ -1,16 +1,25 @@
 # Product Management
 
-We often talk about project managers, and refer to what we do as projects, but *what is a project?*
+We often talk about project managers, and refer to what we do as projects, but _what is a project?_
 
 The [PMI](http://www.pmi.org/) defines a project like this:
-> A project can be defined as a **temporary endeavor undertaken to create a unique product or service.**
 
+> A project can be defined as a **temporary endeavor undertaken to create a unique product or service.**
+>
 > Projects are different from other ongoing operations in an organization, because unlike operations, projects have a definite beginning and an end - they have a **limited duration**.
 
 Others use projects to create the `v1.0` of a new product. We use [MVPs](https://en.wikipedia.org/wiki/Minimum_viable_product).
 
 They use projects to plan and execute every new release. We use [continuous delivery](http://martinfowler.com/bliki/ContinuousDelivery.html) to incrementally bring value to our customers.
 
-In buildo, big projects don't exist. That's why we don't need a lot of traditional project management. The following sections outline our product management workflow, a continuous flow that starts with customer requests and ends with delivered business value.
+In buildo, big projects are not the norm. That's why we don't need a lot of traditional project management. The following sections outline our product management workflow, a continuous flow that starts with customer requests and ends with delivered business value.
+
+**disclaimer: ** we'll still refer to our products as projects, and talk about project managers, that's life. As long as we understand the fundamental difference between a single project and a continuously evolving product, we should be fine.
```

Comment:
se metti gli spazi prima di `**` GitHub li parsa male, non so se succede anche nel book, ma 
`**disclaimer: **` renderizza in **disclaimer: **
mentre `**disclaimer:**` renderizza in **disclaimer:**

Visto che questi sono comunque file markdown hostati su GitHub, suggerisco di rimpiazzare i vari `: **` con `:**` dappertutto